chromium_src/chrome/installer/util/shell_util.cc

```diff
@@ -34,13 +34,6 @@ int GetIconIndexForFileType() {
 
 }  // namespace
 
-#define BRAVE_IPFS L"ipfs"
-#define BRAVE_IPNS L"ipns"
-
-#define BRAVE_BROWSER_PROTOCOL_ASSOCIATIONS BRAVE_IPFS, BRAVE_IPNS,
```

Comment:
should we deassociate these protocols somehow? probably a minor issue anyway

Reply:
It is association of the default app per protocol, it does it for windows with help of COM interface: `IApplicationAssociationRegistration` method: SetAppAsDefault,
but there is no any place where we clean that association, and interface  `IApplicationAssociationRegistration` has method to clean all associations for  current user but not only one.